src/configuration.ts

```diff
@@ -120,72 +121,74 @@ export class Configuration {
         AllowedToolboxItem.ClassCounter,
         AllowedToolboxItem.KeypointSlider,
         AllowedToolboxItem.SubmitButtons,
-    ]
+    ];
 
     public default_keybinds = {
-        "annotation_size_small": "s", //The s Key by default
-        "annotation_size_large": "l", //The l Key by default
-        "annotation_size_plus": "=",   //The = Key by default
-        "annotation_size_minus": "-",  //The - Key by default
-        "annotation_vanish": "v"      //The v Key by default
-    }
+        annotation_size_small: "s", // The s Key by default
+        annotation_size_large: "l", // The l Key by default
+        annotation_size_plus: "=", // The = Key by default
+        annotation_size_minus: "-", // The - Key by default
+        annotation_vanish: "v", // The v Key by default
+    };
```

Comment:
```suggestion
        annotation_size_small: "s",
        annotation_size_large: "l",
        annotation_size_plus: "=",
        annotation_size_minus: "-",
        annotation_vanish: "v",
    };
```
new formatting looks gross and these are obvious anyways

Reply:
I outdated this with my other change, so I fixed this in 56193d6